library/std/src/sys/fs/vexos.rs

```diff
@@ -537,7 +543,9 @@ pub fn copy(from: &Path, to: &Path) -> io::Result<u64> {
     let mut reader = File::open(from)?;
     let mut writer = File::create(to)?;
 
-    io::copy(&mut reader, &mut writer)
+    let ret = io::copy(&mut reader, &mut writer)?;
+    writer.close()?;
```

Comment:
Even though this can't fail, I copied it for symmetry with the other implementations.